src/components/TimePicker/timeSelect.js

```diff
@@ -80,6 +91,12 @@ export default class TimeSelect extends Component {
         });
     }
 
+    handleClickOutside() {
+        if (this.inputFocusedIndex === -1) return;
+        this.inputFocusedIndex = -1;
+        this.forceUpdate();
```

Comment:
Instead of make a force update manually which is not recommended, I think we should put this.inputFocusedIndex in state to make it reactive